.github/workflows/rebuild.yml

```diff
@@ -0,0 +1,49 @@
+name: Rebuild
+
+on:
+  schedule:
+    # 7am (UTC) each day
+    # '*' is a special character in YAML so you have to quote this string
+    - cron:  '0 7 * * *'
+  # allows the workflow to be manually executed any time
+  workflow_dispatch:
+
+jobs:
+  image-refresh:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        java: [ '11' ]
+        tag: [ '4.9.2', '4.10.2' ]
+      fail-fast: false
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        ref: refs/tags/${{ matrix.tag }}
+    - name: Setup java
+      uses: actions/setup-java@v2
+      with:
+        distribution: 'temurin'
+        java-version: ${{ matrix.java }}
+        cache: 'maven'
+    - name: Build and Release
+      env:
+        BASE: origin/${{ github['base_ref'] }}
+        GITHUB_TOKEN: ${{ secrets.DOCS_SITE_TOKEN }}
+        GITHUB_REPOSITORY: ${{ secrets.GITHUB_REPOSITORY }}
+        GITHUB_ACTOR: ${{ secrets.GITHUB_ACTOR }}
+        DOCKERHUB_USERNAME: ${{ secrets.DOCKERHUB_USERNAME }}
+        DOCKERHUB_TOKEN: ${{ secrets.DOCKERHUB_TOKEN }}
+        GPG_PASSPHRASE: ${{ secrets.GPG_PASSPHRASE }}
+        GPG_KEYNAME: ${{ secrets.GPG_KEYNAME }}
+        GPG_KEY_FILE: ${{ secrets.GPG_KEY_FILE }}
+        OSSRH_USERNAME: ${{ secrets.OSSRH_USERNAME }}
+        OSSRH_PASSWORD: ${{ secrets.OSSRH_PASSWORD }}
+        OLD_JAVA_HOME: /opt/hostedtoolcache/AdoptOpenJDK/1.0.0-ga-11-jdk-hotspot-linux-x64-normal-latest/x64/
+      run: |
+        . build/release/global.sh
+        bash build/release/00_prep.sh
+        bash build/release/10_build.sh
+        bash build/release/20_test.sh
+        bash build/release/30_release.sh
+        bash build/release/40_drop.sh
```

Comment:
you'll need to exclude the MavenCentral stuff.

Reply:
it seemed like those were in the later phases (which I removed from this workflow), maybe there's some setup work i missed in one of these...any pointers?

Reply:
ah, i see it now, the first step of 30_release.sh:

```
bash ${WORKSPACE}/build/release/bin/30_release/0_sonatype.sh

# This should always be last
bash ${WORKSPACE}/build/release/bin/30_release/1_dockerhub.sh
```

maybe i can just call `30_release/1_dockerhub.sh` directly from the workflow instead of clalling 30_release.sh?

Reply:
I think that'd be preferrable.

Reply:
done